.github/workflows/gpu_full_multi_gpu_tests.yml

```diff
@@ -49,7 +49,7 @@ jobs:
               - tfhe/src/c_api/**
               - 'tfhe/docs/**.md'
               - Makefile
-              - '.github/workflows/aws_tfhe_multi_gpu**'
+              - '.github/workflows/**_multi_gpu_tests.yml'
```

Comment:
Why the `**`?

Reply:
Ah I didn't write the exact name of the workflow, ** is a placeholder for "any character".